### ISSUE TITLE
Bug Report: Oauth2 Redirect Url malformed

### ISSUE THREAD

**Issue by derocs** [Azure/azure-functions-openapi-extension#419]
I have an a Durable Task Azure function I created in vs2022, using Microsoft.Azure.WebJobs.Extensions.OpenApi v1.3.0
I configured OpenAPI to use OpenApiVersionType.V3.  When I try to Authorize the swagger client, I get the prompt asking for the Client Id and Scope I requested, I provide those, it routes me to Microsoft, but the redirectUrl provided to microsoft is wrong:

redirect_uri=http%3A%2F%2Flocalhost%3A5800https%3A%2F%2Flocalhost%3A5800%2Fapi%2Foauth2-redirect.html

It appears to be combining 2 full absolute urls:

http://localhost:5800 + https://localhost:5800/api/oauth2-redirect.html

Why does it do this?  Is there some configuration option to fix this, or override this behavior?

**Comment by derocs** [comment]
Just bumping this.

**Comment by andrekiba** [comment]
I have the same problem, @justinyoo could you please tell as if there is a temporary fix that we can do?

**Comment by maartendevos** [comment]
Same issue here as well

**Comment by andrekiba** [comment]
yes and this blocks any oauth2 oidc integration...so I don't understand...is someone in charge of this @justinyoo?
Really someone can use this without security? 
There different problems and the delay of answers make me worry...

**Comment by andrekiba** [comment]
it seems also already fixed https://github.com/Azure/azure-functions-openapi-extension/pull/385 but the 1.4.0 is not yet released

**Comment by derocs** [comment]
@chinarosesz , which version did you have to go back to?  I'm not a software dev.  Just trying to figure out which version i can use while i wait.

**Comment by chinarosesz** [comment]
> @chinarosesz , which version did you have to go back to? I'm not a software dev. Just trying to figure out which version i can use while i wait.

I had to go back to version 1.0 :(

**Comment by justinyoo** [comment]
Thanks everyone for the long wait! I've got a couple of items to be merged into v1.4.0, which are on the review. Please wait for a little bit longer!
